weave-js/src/components/PagePanelComponents/Home/HomePreviewSidebar.tsx

```diff
@@ -27,6 +34,18 @@ const CenterTableActionCellIcon = styled(LayoutElements.VStack)`
   }
 `;
 
+const DashboardTemplateItem = styled(LayoutElements.HStack)`
+  padding: 8px 12px;
+  border: 1px solid #dadee3;
+  border-radius: 4px;
+  cursor: pointer;
+  &:hover {
+    border: 1px solid #a9edf2;
+    background-color: #a9edf212;
+    color: #038194;
+  }
+`;
+
```

Comment:
I like throwing a `displayName` on all styled components, I find it a lot easier to figure out what is going on when viewing things in the React developer tools.

Reply:
fixed in next commit